xmlWorksheet.go

```diff
@@ -20,6 +20,7 @@ import (
 // http://schemas.openxmlformats.org/spreadsheetml/2006/main.
 type xlsxWorksheet struct {
 	mu                     sync.Mutex
+	formulaSharedIndex     sync.Map
```

Comment:
This changes will effect the reflect fields calculation. For example, the `from` and `to` offset of [`bulkAppendFields`](https://github.com/xuri/excelize/blob/0f19d7fcd7c7c6ab3f25a363ced93896a4d323e8/stream.go#L723-L733) caller needs to update.